docs/docs/plugins-themes-and-starters.md

```diff
@@ -36,3 +36,83 @@ Gatsby offers [official starters](/docs/starters/#official-starters) for a defau
+| ◐    | Somewhat capable (support is minimal or it is not best practice) |
+| ○    | Not capable                                                      |
+
+### Differences and considerations in maintenance
+
+When it comes to maintaining a Gatsby site, plugins and themes offer a distinct advantage to starters by being distributed as packages. This means making changes in multiple Gatsby sites is as easy as a new install of an updated package upstream. It's difficult to sync changes across multiple sites derived from the same starter.
+
+| Maintenance        | Plugin | Theme | Starter |
+| ------------------ | ------ | ----- | ------- |
+| Versioning         | ●      | ●     | ◐       |
+| Install as Package | ●      | ●     | ○       |
+
+#### Versioning
+
+Starters can still be versioned inside of a repository so that you can track issues or bugs associated with specific updates, but they aren't formally released and published, so they aren't versioned like a plugin or theme is.
+
+#### Installing as a package
+
+Starters can't be installed into existing sites, this problem was one of the motivating factors in developing the newer concept of themes. You can read more about the rationale for themes in the [What are Gatsby Themes guide](/docs/themes/what-are-gatsby-themes/#gatsby-starters).
+
+### Differences and considerations in configuration
+
+Plugins and themes can expose options to make them configurable for users. There are different possibilities for configuration like passing in options and shadowing files that make plugins and themes more powerful—but also more complicated—than starters. Because themes are also plugins, shadowing is possible in plugins as well. It may make sense for a plugin to take advantage of the shadowing API, but it is less common.
+
+| Configuration         | Plugin | Theme | Starter |
+| --------------------- | ------ | ----- | ------- |
+| Pass in Options       | ●      | ●     | ◐       |
+| Shadowing             | ◐      | ●     | ○       |
+| Uses Multiple Plugins | ◐      | ●     | ●       |
+| Custom components     | ◐      | ●     | ●       |
+
+#### Pass in Options
+
+Plugins and themes both allow options to be passed in when installed in the plugins array of a `gatsby-config`. Starters can be set up with documented options for customization, but there are no officially supported options for starters apart from what the author of the starter decides to write.
+
+#### Shadowing
+
+Theme, [shadowing](/docs/themes/shadowing/) exists to allow users to override or otherwise extend individual components provided by a theme. For example, a plugin or theme can provide a specific path to `gatsby-config` so the plugin knows where to build pages from, but the user wouldn't be able adjust _how_ those pages are built, only from what path. Theme shadowing allows users to replace a file with their own version of it, allowing them to rewrite that logic to use the path in a different way.
```

Comment:
```suggestion
Theme [shadowing](/docs/themes/shadowing/) exists to allow users to override or otherwise extend individual components provided by a theme. For example, a plugin or theme can provide a specific path to `gatsby-config` so the plugin knows where to build pages from, but the user wouldn't be able adjust _how_ those pages are built, only from what path. Theme shadowing allows users to replace a file with their own version of it, allowing them to rewrite that logic to use the path in a different way.
```